docs/.vuepress/config.js

```diff
@@ -1,4 +1,21 @@
-var sidebar = require('../api/0.5.0/sidebar')
+const sidebar = require('../api/0.5.0/sidebar')
+const glob = require('glob')
+
+// function for loading all MD files in a directory
+const getChildren = function(parent, path) {
+  return glob
+    .sync(parent + '/' + path + '/**/*.md')
+    .map(f => {
+      // remove "parent" and ".md"
+      f = f.slice(parent.length + 1, -3)
+      // remove README
+      if (f.endsWith('README')) {
+        f = f.slice(0, -6)
+      }
+      return f
+    })
+    .sort()
+}
```

Comment:
@dylanbhughes can I get a consult on this JS code? 😝

Reply:
single character variable names 👎